test/Microsoft.ML.Tests/TrainerEstimators/OnlineLinearTests.cs

```diff
@@ -25,7 +25,7 @@ public void OnlineLinearWorkout()
 
             var trainData = pipe.Fit(data).Transform(data).AsDynamic;
 
-            IEstimator<ITransformer> est = new OnlineGradientDescentTrainer(Env, new OnlineGradientDescentTrainer.Arguments());
+            IEstimator<ITransformer> est = new OnlineGradientDescentTrainer(Env, "Label", "Features");
```

Comment:
>"Label" [](start = 81, length = 7)

Any chance I can convince you to use DefaultColumnNames.Label?

Reply:
I can change, it; it won't be any better than it is. 
This is "Label", because in like 20 we defined r.Label. 

There is no way to type the the member name to be the same as a particular string.
(might be the only thing i miss from javascript :P)

---
In reply to: [221112970](https://github.com/dotnet/machinelearning/pull/1084#discussion_r221112970) [](ancestors = 221112970)